src/vue/modules/issuance-explorer/components/issuances-table.vue

```diff
@@ -80,3 +124,38 @@ export default {
   },
 }
 </script>
+
+<style lang="scss">
+  @import "~@scss/variables";
+  .app__table {
```

Comment:
Please use BEM here. This is unacceptable. If you want to use global styles, we need to discuss it and create some global styles.